grammar.js

```diff
@@ -117,16 +117,31 @@ module.exports = grammar({
       ),
 
     directive: ($) =>
-      seq(
-        choice("<%", "<%=", "<%%", "<%%="),
-        prec.left(
-          seq(
-            choice(
-              $.partial_expression_value,
-              $.ending_expression_value,
-              $.expression_value
-            ),
-            "%>"
+      choice(
+        seq(
+          choice("<%", "<%=", "<%%", "<%%="),
+          prec.left(
+            seq(
+              choice(
+                $.partial_expression_value,
+                $.ending_expression_value,
+                $.expression_value
+              ),
+              choice("%>")
+            )
+          )
+        ),
+        seq(
+          choice("{"),
+          prec.left(
+            seq(
+              choice(
+                $.partial_expression_value,
+                $.ending_expression_value,
+                $.expression_value
+              ),
+              choice("}")
+            )
           )
         )
```

Comment:
Partial/end expression is not allowed in `{...}`, it should always be a single expression. We can do this:

```suggestion
        alias($.expression, "expression")
```

Aliasing as string makes the node anonymous, so it appears as `directive`.

Reply:
Actually, I would consider parsing `{...}` as `expression` node, regardless if it's `<div attr={...}>` or `<div>{...}</div>`. So it would be this:

```diff
_node: ($) =>
-      choice($.doctype, $.tag, $.component, $.text, $.comment, $.directive),
+      choice($.doctype, $.tag, $.component, $.text, $.comment, $.directive, $.expression),
```

@connorlay wdyt?

Reply:
@jonatanklosko That makes sense to me!